config/initializers/commontator.rb

```diff
@@ -1,187 +1,249 @@
 
-  # Proc that is passed the current controller as argument
-  # Returns the current user
+  # current_user_proc
+  # Type: Proc
+  # Arguments: the current controller (ActionController::Base)
+  # Returns: the current user (acts_as_commontator)
+  # The default works for Devise and similar authentication plugins
   # Default: lambda { |controller| controller.current_user }
   config.current_user_proc = lambda { |controller| controller.current_user }
 
-  # Proc called with the view_context object as argument
-  # Returns a string to be appended to all JavaScript responses from commontator
+  # javascript_proc
+  # Type: Proc
+  # Arguments: a view (ActionView::Base)
+  # Returns: a String that is appended to Commontator JS views
   # Can be used, for example, to display/clear Rails error messages
-  # Objects visible in view templates can be accessed through
-  # the view_context object (for example, view_context.flash)
-  # However, the view_context does not include the main application's helpers
-  # Default: lambda { |view_context| '$("#error_explanation").remove();' }
-  config.javascript_proc = lambda { |view_context| '$("#error_explanation").remove();' }
-
-
-  # User (acts_as_commontator) Configuration
-
-  # Whether the comment creator's name is clickable in the comment view
-  # If enabled, the link will point to the comment creator's 'show' page
-  # Default: false
-  config.user_name_clickable = false
-
-  # Proc called with user as argument
-  # Returns the user's name
-  # Default: lambda { |user| 'Anonymous' } (all users are Anonymous)
-  config.user_name_proc = lambda { |user| user.public_name }
-
-  # Proc called with user as argument
-  # Returns the user's email address
-  # Used in the subscription mailer
-  # Default: lambda { |user| user.email }
-  config.user_email_proc = lambda { |user| user.email }
-
-  # Proc called with user as argument
-  # Returns true iif the user is an admin
-  # Admins can delete other users' comments and close threads
-  # Default: lambda { |user| false } (no admins)
-  config.user_admin_proc = lambda { |user| user.admin? }
-  
-  # Proc called with user as argument
-  # Returns true iif the user should receive subscription emails
-  # Default: lambda { |user| true } (always receive subscription emails)
-  config.subscription_email_enable_proc = lambda { |user| true }
-
-
-  # Commontable (acts_as_commontable) Configuration
-
-  # What a comment is called in your application
-  # Default: 'comment'
-  config.comment_name = 'comment'
-
-  # Verb used when creating comments (present)
-  # Default: 'post'
-  config.comment_create_verb_present = 'post'
+  # or to reapply JQuery UI styles after Ajax calls
+  # Objects visible in view templates can be accessed
+  # through the view object (for example, view.flash)
+  # However, the view does not include the main application's helpers
+  # Default: lambda { |view| '$("#error_explanation").remove();' }
+  config.javascript_proc = lambda { |view|
```

Comment:
Unused block argument - `view`. If it's necessary, use `_` or `_view` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.